crates/types/protobuf/restate/cluster.proto

```diff
@@ -21,6 +21,7 @@ message ClusterState {
   google.protobuf.Duration last_refreshed = 1;
   restate.common.Version nodes_config_version = 2;
   restate.common.Version partition_table_version = 3;
+  restate.common.Version logs_metadata_version = 5;
```

Comment:
OCD: I believe we can swap the proto field ids to keep them in order, as we don't have substantial (any?) usage of restatectl yet, nevermind across versions?

Reply:
I'd rather keep them ordered by tag, it makes it easy to spot gaps. In fact, the common convention when removing a deprecated one is to leave a comment in-situ with the tag and why it's removed.